nova/virt/vmwareapi/vmops.py

```diff
@@ -2221,6 +2222,53 @@ def manage_image_cache(self, context, instances):
             datastores_info.append((ds, dc_info))
         self._imagecache.update(context, instances, datastores_info)
 
+        if CONF.vmware.image_as_template:
+            self._age_cached_image_templates(dc_info, instances)
```

Comment:
I just noticed, that we still have templates in {{qa-de-1}} at least, but we don't have that setting enabled anymore ... so they will never go away ... Do you think it would make sense to age the images unconditionally or should I remove them by hand?

Reply:
I don't like the idea of removing the condition and having the cleanup happening no matter whether the image-as-template is enabled or not. Still, if you prefer unconditional cleanup, we'll be fine changing it.

Reply:
Would it be easy to trigger the cleanup manually? Do you think it would be feasible via backdoor shell?

Reply:
Backdoor should be doable but coding a separate script mimicking the cleanup seems simpler and cleaner.